azure-devops/azext_devops/dev/common/_credentials.py

```diff
@@ -29,18 +32,46 @@ def set_credential(devops_organization, token):
     cred_store = CredentialStore()
     cred_store.set_password(key, token)
 
+    if os.path.isfile(_ORGANIZATION_LIST_FILE):
+        # No need to add organization if it's already present.
+        with open(_ORGANIZATION_LIST_FILE, 'r') as org_list:
+            for organization in org_list:
+                if key == organization.rstrip():
+                    return
+    with open(_ORGANIZATION_LIST_FILE, 'a+') as org_list:
+        org_list.write(key + "\n")
```

Comment:
\n is platform specific. Some platforms have \r\n. 
I think if you use writeline method instead of write you wont need the \n ?

Reply:
Python actually takes care of translating new line to platform specific line separator. [Documentation here](https://docs.python.org/3/library/os.html#os.linesep)

@gauravsaralMs